event_registration_mass_confirm/wizard/confirm_attendees.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, api, _
+
+
+class EventRegistrationMassConfirm(models.TransientModel):
+
+    _name = "event.registration.massconfirm"
+
+    @api.multi
+    def confirm_registrations(self):
+        """Validate selected registrations."""
+        context = dict(self.env.context or {})
+        active_ids = context.get('active_ids', []) or []
```

Comment:
`context.get('active_ids') or []`

Reply:
then, if empty it means that the user has not selected anything: raise a user error